ros_buildfarm/templates/status/release_compare_page.html.em

```diff
@@ -14,7 +15,12 @@
   <link rel="stylesheet" type="text/css" href="css/compare_page.css?@(resource_hashes['compare_page.css'])" />
 </head>
 <body>
-  <script type="text/javascript">window.body_ready();</script>
+@{
+import time
```

Comment:
`start_time` is a formatted string. I'm expecting a unix timestamp in floating point seconds.